### PR TITLE
Add option to set a default destination

### LINKED ISSUE #4079: Add option to select a default destination
The default etcd backup, which is created for each cluster, didn't work when using the new backup destinations feature, as it didn't set a destination. This is caused, because the backup doesn't know which destination it should use.

Therefor we need to add an option to select a default destination, which then will be used for the default etcd backup.

### PR DESCRIPTION
**Pull request by kgroschoff**
### What this PR does / why we need it
Add option to set a default destination. It'll then be used for automatic etcd backups after cluster creation.

### Which issue(s) this PR fixes

Fixes #4079

### Special notes for your reviewer

defaultdestination1

defaultdestination2

### Release note

```release-note
Add option for administrators to specify default destination in "Backup Destinations" view.
```

### CODE REVIEW

**Comment by kgroschoff** [comment]
PTAL @maciaszczykm @floreks

File: src/app/settings/admin/bucket-settings/destinations/component.ts [hunk 1]
```diff
@@ -77,6 +77,13 @@ export class DestinationsComponent implements OnInit {
     return !!destination?.credentials;
   }
 
+  isDefault(name: string): boolean {
+    return (
+      this.seed?.spec?.etcdBackupRestore?.defaultDestination &&
+      this.seed.spec.etcdBackupRestore.defaultDestination === name
```
Comment (maciaszczykm) [inline src/app/settings/admin/bucket-settings/destinations/component.ts:83]: `this.seed?.spec?.etcdBackupRestore?.defaultDestination === name`.

Comment (kgroschoff) [reply src/app/settings/admin/bucket-settings/destinations/component.ts]: right. fixed 👍

**Pull request merged**